efficientdet/aug/mosaic_test.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+Mosaic Augmentation simple test.
+"""
+from absl import logging
+import tensorflow.compat.v1 as tf
+from aug.mosaic import Mosaic
+
+
+class MosaicTest(tf.test.TestCase):
+
+    def __init__(self,*args,**kwargs):
+        super().__init__(*args,**kwargs)
+        self.output_size = (512,512)
+        self.mosaic = Mosaic(out_size=self.output_size)
+        tf.random.set_random_seed(111111)
+
+    def test_mosaic_boxes(self):
```

Comment:
Can you add a test that uses 4 tiny images?  Ie 4x4 images of all zeroes, ones, twos, etc.

Reply:
@LucasSloan Sure i can

Reply:
Also add a test for the bounding box munging.

Reply:
@LucasSloan can you add munging test.

Reply:
added tiny image test.